src/style.css

```diff
@@ -1 +1,112 @@
-/* add styles here */
+html {
+  --main-color: var(--main-color);
```

Comment:
Why did you add this style? 

it isn't good practice main-color for `html` tag